prow/entrypoint/run_test.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package entrypoint
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"k8s.io/test-infra/prow/pod-utils/wrapper"
+)
+
+func TestOptions_Run(t *testing.T) {
+	var testCases = []struct {
+		name           string
+		args           []string
+		expectedLog    string
+		expectedMarker string
+		expectedErr    bool
+	}{
+		{
+			name:           "successful command",
+			args:           []string{"/usr/bin/sh", "-c", "exit 0"},
```

Comment:
is this a macism? I'm used to things like `sh` and `echo` being in `/bin`, not `/usr/bin`.

alternately, can we rely on using `$PATH` to resolve these?

Reply:
This may be a Red-Hat-ism. IIRC `exec.Command` doesn't make use of the `$PATH`/

Reply:
Hmm NVM, `$PATH` resolution seems to work